packages/cli/src/commands/setup/deploy/providers/aws-serverless.js

```diff
@@ -24,19 +24,37 @@ custom:
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
   region: us-east-2 # This is the AWS region where the service will be deployed.
   httpApi: # HTTP API is used by default. To learn about the available options in API Gateway, see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html
-    cors: true
+    cors:
+      allowOrigins:
+        - '*' # This is the default value. You can remove this line if you want to restrict the CORS to a specific origin.
```

Comment:
Hey @virtuoushub - I'm back :) 

I'm getting this error when trying to deploy, maybe there's something a little bit wrong? 

```
Serverless: DOTENV: Loading environment variables from .env:
Serverless: Configuration warning at 'provider.httpApi.cors': unrecognized property 'allowOrigins'
Serverless:
```

Reply:
Nevermind, fixed it `allowOrigins` -> `allowedOrigins`